src/shared/const/routes.ts

```diff
@@ -19,66 +19,77 @@ const routes = [
     component: MainPage,
     title: 'Старт',
     exact: true,
+    isProtected: false,
```

Comment:
🧐
Это необязательно, но)
Переменные хочется называть `isSmth` потому, что ты не знаешь какую-то информаци заранее и вычисляешь по месту + эта информация не константна, может меняться и ты как бы спрашиваешь: "Ок, это так или нет в _данный момент времени_?"
А тут это конфиг, он константен. Тут вопрос другой: "Ок, я считаю этот роут приватным или нет?". Можно взять в пример `exact`, он не написан как `isExact`
```suggestion
    protected: false,
```